readthedocs/proxito/views/serve.py

```diff
@@ -243,6 +242,39 @@ def get(self,
             filename=filename,
         )
 
+    def _get_canonical_redirect_type(self, request):
+        """If the current request needs a redirect, return the type of redirect to perform."""
```

Comment:
Isn't this method missing the `/` -> `/en/latest` (`RedirectType.system`) redirect?

Reply:
Hrm... That's happening in another view, I guess, tho.

Reply:
It's in the same view, but more down, after doing some other checks, we have a specific method for that at https://github.com/readthedocs/readthedocs.org/blob/0912c0c998093546c91d48bc0d26f8dbfa8e088a/readthedocs/proxito/views/mixins.py#L301-L303